backend/apps/slack/management/commands/slack_sync_data.py

```
@@ -42,89 +34,114 @@ def handle(self, *args, **options):
-
-                    cursor = response.get("response_metadata", {}).get("next_cursor")
-                    if not cursor:
-                        break
+                response = func(*args, **kwargs)
+                if not response.get("ok", False):
+                    raise SlackApiError(f"{func.__name__} failed", response)
+                return response
             except SlackApiError as e:
-                self.stdout.write(
-                    self.style.ERROR(f"Failed to fetch members: {e.response['error']}")
+                if e.response.status_code == 429:  # Rate limit hit
```

Ruff (0.11.9)

Magic value used in comparison, consider replacing `429` with a constant variable

(PLR2004)